### PR TITLE
Send email to trainee when assigned to course or removed from course

### PR DESCRIPTION
**Pull request by QuangVinh93**

### CODE REVIEW

**Comment by hitshootnum1** [comment]
ok

**Comment by HUSTToanLH** [comment]
ok

File: app/models/user_course.rb [hunk 1]
```diff
@@ -1,6 +1,17 @@
 class UserCourse < ActiveRecord::Base
+  after_create :send_email_when_assigned
+  after_destroy :send_email_when_removed
+
   belongs_to :user
   belongs_to :course
 
   delegate :name, :start_course, to: :user, prefix: true
+
+  def send_email_when_assigned
```
Comment (dieunb) [inline app/models/user_course.rb]: private

File: app/models/user_course.rb [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+    TraineeWorker.perform_async TraineeWorker::ASSIGN_TRAINEE, self.user_id, self.course_id
+  end
+
+  def send_email_when_removed
```
Comment (dieunb) [inline app/models/user_course.rb]: private

**Pull request merged**